### DISCUSSION TITLE
First time working with Express & Supabase ... help 😀

### DISCUSSION THREAD

**Discussion by rogersm-uwosh** [supabase/supabase#28826] [Questions]
I've had good luck working with Supabase and Flutter previously, but working with Supabase and Express I'm having some conceptual issues. I want to be able to handle multiple users.  If I just call supabase.auth.signInWithPassword(), it works, but as soon as somebody else logs in they're now the user. So, I decided to a) stash the jwt returned from signInWithPassword() in an Express session, and b) recreate the client whenever I perform a SQL operation.

Is there a better way to do this? (Probably). Is there any documentation that covers this? I couldn't find any examples pertaining to Express in the Supabase docs.

**Comment by GaryAustin1** [comment]
https://supabase.com/docs/guides/getting-started/tutorials/with-expo-react-native
https://supabase.com/docs/guides/auth/quickstarts/react-native
I can't hope with RN/Expo, but a couple of links.
There are a few expo users on the Supabase Discord also.

**Comment by silentworks** [comment]
There is express example code here https://supabase.com/docs/guides/auth/server-side/creating-a-client?queryGroups=framework&framework=express&queryGroups=environment&environment=server-client. There is also this codebase I created which uses express https://github.com/silentworks/supabase-by-example/tree/main/express
